examples/README.md

```diff
@@ -1,49 +1,89 @@
 # Smithy Rust Server SDK examples
 
-This folder contains an example services showcasing the service framework capabilities and to run benchmarks.
+This folder contains an example services showcasing Smithy Rust Server SDK, 
+also known as the Rust service framework, capabilities and to run benchmarks.
 
-- `/pokemon-service`, a HTTP server implementation demonstrating [middleware](https://awslabs.github.io/smithy-rs/design/server/middleware.html)
-and [extractors](https://awslabs.github.io/smithy-rs/design/server/from_parts.html).
-- `/pokemon-service-tls`, a minimal HTTPS server implementation.
-- `/pokemon-service-lambda`, a minimal Lambda deployment.
+Three server implementations are available:
 
-The `/{binary}/tests` folders are integration tests involving the generated clients.
+- `/pokemon-service`, a HTTP server demonstrating [middleware] and [extractors].
+- `/pokemon-service-tls`, a HTTPS server. This server can do 
+   its own TLS negotiation, rather than relying on a load balancer. 
+- `/pokemon-service-lambda`, a server that can be deployed onto AWS Lambda.
 
-## Build
+These servers, and their clients, are generated using smithy-rs. You're invited
+to benchmark the performance of these servers to see whether smithy-rs might be
+a suitable choice for implementing your web service.
 
-Since this example requires both the server and client SDK to be code-generated
+[middleware]: https://awslabs.github.io/smithy-rs/design/server/middleware.html
+[extractors]: https://awslabs.github.io/smithy-rs/design/server/from_parts.html
+
+
+## Pre-requisites
+
+You will need install Java 11 (or higher) and [Gradle](https://gradle.org/) 7 
+(or higher) to run the smithy-rs code generator. You will need a working 
```

Comment:
Gradle is provisioned automatically at the right version when you run `gradlew` the first time.

Reply:
Also, the compatibility range is actually limited—I'd probably say "Java 11" and be done—it works with a few other versions but not all versions

Reply:
Thanks for taking a look, both. Fixed here https://github.com/awslabs/smithy-rs/pull/2570/commits/6f43a853537d2ac3d2998303f84677a2c42a5d77